.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Login.gov Issues
+    url: https://www.login.gov/contact/
+    about: For help with Login.gov
+  - name: Report a vulnerability
+    url: https://github.com/18F/handbook/security/advisories/new
+    about:  Privately report a security vulnerability.
```

Comment:
This section is already present, but I cannot find where it's defined, so I'm including it in this PR to make sure it's not lost.

Reply:
The URL for "Report a Vulnerability" should point to the GSA disclosure policy "https://www.gsa.gov/vulnerability-disclosure-policy"

Reply:
@alxhrck Interesting. That's not where it goes today. It's easy enough for me to change it here, but I feel like it sort of deserves its own top-level PR vs. being a side-effect of this change?

Reply:
That's a good suggestion. I'll make that update a separate PR.